crowbar_framework/app/helpers/barclamp/monasca_helper.rb

```diff
@@ -27,5 +27,21 @@ def log_levels(selected)
         selected.to_s
       )
     end
+
+    def api_log_levels(selected)
+      options_for_select(
+        [
+          ["CRITICAL", "CRITICAL"],
+          ["FATAL", "FATAL"],
+          ["ERROR", "ERROR"],
+          ["WARNING", "WARNING"],
+          ["INFO", "INFO"],
+          ["DEBUG", "DEBUG"],
+          ["NOTSET", "NOTSET"],
```

Comment:
I'd remove **NOTSET** level, as nobody (;) ) knows what it means. (Isn't it a kind of internal level?)